actions/docker-publish/action.yaml

```diff
@@ -0,0 +1,60 @@
+name: "Publish docker image"
+description: "Publish docker image from download artifact"
+
+inputs:
+  image-name:
+    descritption: "Name of docker image (Default is the respository name)."
+    required: true
+    default: ${{ github.event.repository.name }}
+  image-artifact-name:
+    descritption: "Name of the pipeline artifact that contains the docker image.tar file to push, see https://github.com/actions/upload-artifact (Default is 'docker-image')."
+    required: true
+    default: "docker-image"  
+  publisher:
+    description: "Publisher to prefix docker image (e.g. 'my-publisher' with a repository name)."
+    required: false
+  username:
+    description: "Username for the docker registry login."
+    required: true
+  password:
+    description: "Password for the docker registry login."
+    required: true
+  working-directory:
+    description: "Working directory of your Gradle artifacts. (Default is root directory)"
+    required: false
+    default: "./"
+runs:
+  using: "composite"
+  steps:
+    - name: Download docker image tar artifact
+      uses: actions/download-artifact@master
+      with:
+        name: ${{ inputs.image-artifact-name }}
+        path: build
```

Comment:
Don't you have to include the `working-directory` here as well?

Reply:
I don't think so, because the action downloads an artifact and publishes using the `docker` command, so the folder should not matter, right?

Reply:
My point is that you use `build/*.tar/`  in line 48. Because you set the working directory there, the full path is `~/working-dir/build/*.tar`. If you don't set the working directory here however, the artifact would be in `~/build/`.